README.md

```diff
@@ -38,7 +39,14 @@
 - React
 - Redux
 ### Key Features <a name="key-features"></a>
-- [Live Demo Link](https://space-2cht.onrender.com)
+
+- The project is a single-page application (SPA) built with React and Redux.
+- The data retrieved from the [API](https://api-football-standings.azharimm.dev/) is stored in the Redux store.
+- Each page has a unique route
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo](https://soccerhub.onrender.com/)
```

Comment:
- [ ] Please add in the README a recorded video presenting
- The features of the project
- A piece of the code you find interesting or proud of making